Neo4j.Driver/Neo4j.Driver/Internal/Protocol/IBoltProtocol.cs

```diff
@@ -49,7 +51,6 @@ internal interface IBoltProtocol
         string database,
         Bookmarks bookmarks,
         TransactionConfig config,
-        string impersonatedUser,
```

Comment:
we want impersonation here

Reply:
The impersonated user is in the session config.